ecommerce/serializers.py

```diff
@@ -107,6 +118,120 @@ class Meta:
         model = models.ProductVersion
 
 
+class LineSerializer(serializers.ModelSerializer):
+    """ Line Serializer """
+
+    product = serializers.SerializerMethodField()
+
+    def get_product(self, instance):
+        """ Get the line product id"""
+        return instance.product_version.product.id
+
+    class Meta:
+        fields = ("id", "product")
+        model = models.Line
+
+
+class OrderSerializer(serializers.ModelSerializer):
+    """ Order Serializer """
+
+    name = serializers.SerializerMethodField()
+    close_date = serializers.SerializerMethodField(allow_null=True)
+    amount = serializers.SerializerMethodField()
+    discount_amount = serializers.SerializerMethodField()
+    coupon_code = serializers.SerializerMethodField(allow_null=True)
+    company = serializers.SerializerMethodField(allow_null=True)
+    line_items = serializers.SerializerMethodField()
```

Comment:
Should be able to use `ListField(child=LineSerializer)` instead here: https://www.django-rest-framework.org/api-guide/fields/#listfield

Reply:
switched to `lines = LineSerializer(many=True)`